Makefile

```diff
@@ -69,15 +62,15 @@ unittest: $(GENERATED)
 test: lint copyright unittest
 
 .PHONY: isort
-isort:
+isort: $(GENERATED)
 	pre-commit run isort --all-files
```

Comment:
Do we still need isort?  Ruff sorts imports.

Reply:
Good idea, just removing isort b/c the current ruff config should cover sorting AFAICT